os_net_config/impl_nmstate.py

```diff
@@ -337,9 +401,12 @@ def __dump_key_config(self, config, msg="Applying config"):
         logger.info(f"{msg}\n{cfg_dump}")
 
     def get_vf_config(self, sriov_vf):
-        """Identify the nmstate schema for the given VF
+        """Create the nmstate schema for the given VF
```

Comment:
Fn. definition says "get" - but is it actually "set" ?

Reply:
Yes, may be in another PR, we could fix it